server/src/main/resources/org/jboss/as/server/controller/descriptions/LocalDescriptions.properties

```diff
@@ -73,19 +73,19 @@ server.restart.start-mode=The mode the server should restarted in, can be either
 server.restart.reply=The status of the server following execution of this operation.
 server.stop=Stop a currently running server.
 server.stop.blocking=Whether the operation should block and wait until the server is stopped.
-server.stop.suspend-timeout=If this timeout is set a graceful shutdown will be attempted. If this is zero (the default) then the server will shutdown immediately. A value larger than zero means the server will wait up to this many seconds for all active requests to finish. A value smaller than zero means that the server will wait indefinitely for all active requests to finish.
+server.stop.suspend-timeout=The graceful shutdown timeout in seconds. If this is zero (the default) then the server will shutdown immediately. A value larger than zero means the server will wait up to this many seconds for all active requests to finish. A value smaller than zero means that the server will wait indefinitely for all active requests to finish.
```

Comment:
Shall we prefer to use `stop` word instead of `shutdown` in this case?

Reply:
It makes sense to me, changed, thanks